pkg/adapter/cronjobevents/adapter.go

```diff
@@ -24,13 +24,10 @@ import (
 	"github.com/robfig/cron"
 	"go.uber.org/zap"
 	sourcesv1alpha1 "knative.dev/eventing/pkg/apis/sources/v1alpha1"
-	"knative.dev/eventing/pkg/kncloudevents"
 	"knative.dev/pkg/logging"
 	"knative.dev/pkg/source"
 )
 
-// TODO: this should be a k8s cron.
```

Comment:
Why remove this TODO?

Reply:
open a bug tracking this: https://github.com/knative/eventing/issues/2025